api/src/wfl/api/routes.clj

```diff
@@ -138,7 +137,7 @@
   [status message exception {:keys [uri] :as request}]
   (let [response (exception-handler status message exception request)]
     (log/errorf "Server %s error at occurred at %s :" (:status response) uri)
-    (logr/error exception (:body response))
+    (log/error exception (:body response))
```

Comment:
I imagine there was a reason for using the readable logger initially:

https://github.com/clojure/tools.logging/blob/master/src/main/clojure/clojure/tools/logging/readable.clj

"Logging macros that support printing message (some) arguments as
  if wrapped in pr-str, the goal being to preserve their data representation
  distinct from the explanatory text."
  
  Does the new logger take care of this or should the argument be wrapped in `pr-str`?

Reply:
I suppose in this case it does not matter since there is a call to log/errorf right above it that will create the json log, but pr-str would print to output as a string instead of json which likely would cause stackdriver to not pick it up.

Reply:
I guess it would be helpful to see concrete examples of these error logs and hear from whoever added them and/or has consumed them in the past.